src/main/java/org/fcrepo/importexport/importer/Importer.java

```diff
@@ -167,7 +196,13 @@ public void run() {
         importDirectory(importContainerDirectory);
 
         importMembershipResources();
-        importLogger.info("Finished import... {} resources imported", successCount.get());
+
+        // loop through for nested member/membership resources
+        if (relatedResources.size() > 0) {
```

Comment:
It seems like this block of code can be moved into the `importMemberResources()` method, and called from here.

...and the `importMemberResources()` method should probably be renamed to `importRelatedResources()`.